hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/evm/properties/ModularizedPropertiesTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.web3.evm.properties;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.hedera.mirror.web3.Web3IntegrationTest;
+import com.hedera.node.config.data.ContractsConfig;
+import com.swirlds.config.api.ConfigData;
+import java.lang.reflect.Field;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import org.junit.jupiter.api.Test;
+
+@RequiredArgsConstructor
+class ModularizedPropertiesTest extends Web3IntegrationTest {
```

Comment:
Agree with Bilyana. The standard is if there's both unit and integration to name it with `IntegrationTest` suffix.

```suggestion
class MirrorNodeEvmPropertiesIntegrationTest extends Web3IntegrationTest {
```

Reply:
Comment addressed